crypto not found

**Describe the bug**
When building using the edge runtime (local and on cloudflare workers) I get an error about crypto not being found

**To Reproduce**
Steps to reproduce the behavior:
* Building without edge runtime exported all builds fine. Build with edge runtime set locally and it fails... Build on cloudflare workers results in error too. 

**Expected behavior**
A clear and concise description of what you expected to happen.

**Screenshots**
N/A

**Desktop (please complete the following information):**
 - OS: MacOS
 - Browser N/A
 - authkit-nextjs version 1.1.0
 - Next.js version 14.2.20

**Additional context**

workos-inc/node version - 7.37.0

You can see from the log its pulling in index.worker.js so it knows its on an edge runtime... but then `./node_modules/@workos-inc/node/lib/common/crypto/index.js` pulls in `NodeCryptoProvider.js` and results in Can't resolve crypto... 

See more below for when I upgrade to next.js 15

```

▲  ▲ Next.js 14.2.20
--
12:38:21.205 | ▲  - Experiments (use with caution):
12:38:21.205 | ▲    · turbo
12:38:21.264 | ▲  Creating an optimized production build ...
12:38:40.533 | ▲  Failed to compile.
12:38:40.533 | ▲  ./node_modules/@workos-inc/node/lib/common/crypto/NodeCryptoProvider.js:36:1
12:38:40.534 | ▲  Module not found: Can't resolve 'crypto'
12:38:40.534 | ▲
12:38:40.534 | ▲  https://nextjs.org/docs/messages/module-not-found
12:38:40.534 | ▲
12:38:40.534 | ▲  Import trace for requested module:
12:38:40.534 | ▲  ./node_modules/@workos-inc/node/lib/common/crypto/index.js
12:38:40.535 | ▲  ./node_modules/@workos-inc/node/lib/actions/actions.js
12:38:40.535 | ▲  ./node_modules/@workos-inc/node/lib/index.worker.js
12:38:40.535 | ▲  ./node_modules/@workos-inc/authkit-nextjs/dist/esm/workos.js
12:38:40.535 | ▲  ./node_modules/@workos-inc/authkit-nextjs/dist/esm/actions.js
12:38:40.535 | ▲  ./node_modules/next/dist/build/webpack/loaders/next-flight-action-entry-loader.js?actions=%5B%5B%22%2Fopt%2Fbuildhome%2Frepo%2Fnode_modules%2F%40workos-inc%2Fauthkit-nextjs%2Fdist%2Fesm%2Factions.js%22%2C%5B%22refreshAuthAction%22%2C%22%24%24ACTION_4%22%2C%22%24%24ACTION_0%22%2C%22getAuthAction%22%2C%22handleSignOutAction%22%2C%22getOrganizationAction%22%2C%22%24%24ACTION_1%22%2C%22%24%24ACTION_3%22%2C%22checkSessionAction%22%2C%22%24%24ACTION_2%22%5D%5D%5D&__client_imported__=true!
12:38:40.535 | ▲
12:38:40.535 | ▲  ./node_modules/@workos-inc/node/lib/common/crypto/NodeCryptoProvider.js:36:1
12:38:40.535 | ▲  Module not found: Can't resolve 'crypto'
12:38:40.535 | ▲
12:38:40.536 | ▲  https://nextjs.org/docs/messages/module-not-found
12:38:40.536 | ▲
12:38:40.536 | ▲  Import trace for requested module:
12:38:40.536 | ▲  ./node_modules/@workos-inc/node/lib/common/crypto/index.js
12:38:40.536 | ▲  ./node_modules/@workos-inc/node/lib/actions/actions.js
12:38:40.536 | ▲  ./node_modules/@workos-inc/node/lib/index.worker.js
12:38:40.536 | ▲  ./node_modules/@workos-inc/authkit-nextjs/dist/esm/workos.js
12:38:40.536 | ▲  ./node_modules/@workos-inc/authkit-nextjs/dist/esm/authkit-callback-route.js
12:38:40.536 | ▲  ./node_modules/@workos-inc/authkit-nextjs/dist/esm/index.js


```

When I upgrade to next.js 15....

I get this error

```
○ Compiling /my-boards/settings/[boardId] ...
 ✓ Compiled /my-boards/settings/[boardId] in 2.7s
 ⨯ Error [ReferenceError]: __import_unsupported is not defined
    at <unknown> (.next/server/edge/chunks/ssr/edge-wrapper_3d10d0.js:722)
 ⨯ Error [ReferenceError]: __import_unsupported is not defined
    at <unknown> (.next/server/edge/chunks/ssr/edge-wrapper_3d10d0.js:722)
 ○ Compiling /_error ...
 ✓ Compiled /_error in 665ms
 GET /my-boards/settings/d3a3380280d5cf8c 500 in 3714ms
 ⨯ Error [ReferenceError]: __import_unsupported is not defined
    at <unknown> (.next/server/edge/chunks/ssr/edge-wrapper_3d10d0.js:722)
 ⨯ Error [ReferenceError]: __import_unsupported is not defined
    at <unknown> (.next/server/edge/chunks/ssr/edge-wrapper_3d10d0.js:722)
 GET /my-boards/settings/d3a3380280d5cf8c 500 in 70ms
```

The onscreen error gives me:

```
Server Error

ReferenceError: __import_unsupported is not defined

This error happened while generating the page. Any console logs will be displayed in the terminal window.
<unknown>
[turbopack]/browser/runtime/base/dev-base.ts (236:20)
[project]/node_modules/@workos-inc/node/lib/common/crypto [app-edge-rsc] (ecmascript)
node_modules/@workos-inc/node/lib/common/crypto (1:1)
<unknown>
[turbopack]/browser/runtime/base/dev-base.ts (205:21)
runModuleExecutionHooks
[turbopack]/browser/runtime/base/dev-base.ts (264:5)
instantiateModule
[turbopack]/browser/runtime/base/dev-base.ts (203:5)
getOrInstantiateModuleFromParent
[turbopack]/browser/runtime/base/dev-base.ts (132:10)
commonJsRequire
[turbopack]/shared/runtime-utils.ts (241:18)
[project]/node_modules/@workos-inc/node/lib/common/crypto/NodeCryptoProvider.js [app-edge-rsc] (ecmascript)
node_modules/@workos-inc/node/lib/common/crypto/NodeCryptoProvider.js (36:16)
<unknown>
[turbopack]/browser/runtime/base/dev-base.ts (205:21)
runModuleExecutionHooks
[turbopack]/browser/runtime/base/dev-base.ts (264:5)
instantiateModule
[turbopack]/browser/runtime/base/dev-base.ts (203:5)
Hide ignored frames
```

--- a/package.json
+++ b/package.json
@@ -34,7 +34,7 @@
     "format": "prettier \"{src,__tests__}/**/*.{js,ts,tsx}\" --write"
   },
   "dependencies": {
-    "@workos-inc/node": "^7.37.0",
+    "@workos-inc/node": "^7.37.1",
     "iron-session": "^8.0.1",
     "jose": "^5.2.3",
     "path-to-regexp": "^6.2.2"

